pouch/pouch_utils.go

```diff
@@ -266,10 +266,13 @@ func toHostConfig(hostconfig *containertypes.HostConfig) (*pouchtypes.HostConfig
 		ExtraHosts: hostconfig.ExtraHosts,
 		GroupAdd:   hostconfig.GroupAdd,
 		// InitScript
-		IpcMode:     string(hostconfig.IpcMode),
-		Isolation:   string(hostconfig.Isolation),
-		Links:       hostconfig.Links,
-		LogConfig:   &pouchtypes.HostConfigAO0LogConfig{},
+		IpcMode:   string(hostconfig.IpcMode),
+		Isolation: string(hostconfig.Isolation),
+		Links:     hostconfig.Links,
+		LogConfig: &pouchtypes.LogConfig{
+			LogDriver: hostconfig.LogConfig.Type,
```

Comment:
For this part, I invite @fuweid to take a review at this.

Reply:
If the `LogConfig.Type` is empty, I think we should check the `json` file exists or not. Otherwise, the container failed to restart by invalid log config type.